### DISCUSSION TITLE
[Plesk] WebService is not working properly

### DISCUSSION THREAD

**Discussion by lfernandes** [chatwoot/chatwoot#7788] [Q&A]
### Describe the bug

I have deployed Chatwoot as a Docker container inside a webhost managed by Plesk.

All the system is working well, except for the WebSocket. The console keeps logging an error: `WebSocket connection to 'wss://mysite.com/cable' failed:`

Here is my nginx config:
location /cable {
	proxy_pass http://0.0.0.0:43321;
	proxy_http_version 1.1;
	proxy_set_header Upgrade $http_upgrade;
	proxy_set_header Connection "upgrade";
	proxy_set_header Host $host;
	proxy_set_header X-Real-IP $remote_addr;
	proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
	proxy_set_header X-Forwarded-Proto $scheme;
}

### To Reproduce

1. Install Linux
2. Install Plesk
3. Create a docker instance of Chatwoot
4. Use Plesk UI to create your domain
5. Add in "Docker Proxy Rules" your Chatwoot container

### Expected behavior

Have the websocket working properly to keep the website live udpated

### Environment

Docker

### Cloud Provider

None

### Platform

Browser

### Operating system

Ubuntu 20.04.6 LTS

### Browser and version

Any (Chrome Version 115.0.5790.170 (Official Build) (arm64))

### Docker (if applicable)

Docker version
Docker version 20.10.23, build 7155243

Docker compose
docker-compose version 1.29.2, build 5becea4c
docker-py version: 5.0.0
CPython version: 3.7.10
OpenSSL version: OpenSSL 1.1.0l  10 Sep 2019

**Comment by pranavrajs** [comment]
Moving to Q&A as we don't officially support Plesk.
